controller/templates/confd_settings.py

```diff
@@ -45,3 +45,16 @@
 WEB_ENABLED = bool({{ getv "/deis/controller/webEnabled" }})
 {{ end }}
 UNIT_HOSTNAME = '{{ if exists "/deis/controller/unitHostname" }}{{ getv "/deis/controller/unitHostname" }}{{ else }}default{{ end }}'
+
+# AUTH
+# LDAP
+{{ if exists "/deis/controller/auth/ldap/endpoint" }}
+LDAP_ENDPOINT = '{{ if exists "/deis/controller/auth/ldap/endpoint" }}{{ getv "/deis/controller/auth/ldap/endpoint"}}{{ else }} {{ end }}'
```

Comment:
Oops, sorry I missed this comment before merging. Should we typo-fix the space change?